readme.md

```diff
@@ -204,6 +204,45 @@ Type: `string`
 
 Click the DOM element matching the given [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).
 
+##### inset
+
+Type: `number` or `object`
+
```

Comment:
There should be a description of what the option is here.

Reply:
It should also mention that it both works with and without the `element` option.

Reply:
description added.